Source/Blazorise/Components/ColorEdit/ColorEdit.razor.cs

```diff
@@ -70,13 +70,7 @@ protected override Task<ParseValue<string>> ParseValueFromStringAsync( string va
         }
 
         /// <inheritdoc/>
-        public virtual void Select( bool focus = true )
-        {
-            InvokeAsync( () => SelectAsync( focus ) );
-        }
-
-        /// <inheritdoc/>
-        public virtual async Task SelectAsync( bool focus = true )
+        public virtual async Task Select( bool focus = true )
         {
             await JSRunner.Select( ElementRef, ElementId, focus );
```

Comment:
Might choose to not set up the state machine in the methods from this commit, like in the code base.

Reply:
We can return without await, and add AsTask.